src/FamilyHubs.Referral.Web/Pages/ProfessionalReferral/Confirmation.cshtml

```diff
@@ -15,11 +15,11 @@
             </div>
         </div>
 
-        <p>You have given the family's details to the service. This request has been added to your dashboard and you will receive an email confirmation.</p>
```

Comment:
Could you please replace the target and rel attributes with the new-tab attribute (and remove the '(opens in a new tab)' as that'll get added).